packages/langium/src/parser/lexer.ts

```diff
@@ -21,6 +22,7 @@ export interface LexerResult {
      */
     hidden: IToken[];
     errors: ILexingError[];
+    report?: LexingReport;
```

Comment:
Personally, I would mark `errors` as `@deprecated` and include them in the report. They can point to the same array for backwards compatibility, only to be remove in the next major version.
Also, it would be great if you can add JSDoc